app/src/main/kotlin/com/wire/android/ui/home/HomeScreen.kt

```diff
@@ -31,6 +36,9 @@ import com.wire.android.ui.common.topappbar.search.AppTopBarWithSearchBar
 fun HomeScreen(startScreen: String?, viewModel: HomeViewModel) {
     viewModel.checkRequirements()
     val homeState = rememberHomeState()
+    val coroutineScope = rememberCoroutineScope()
+    var isFileSharingEnabled = false
```

Comment:
we are mixing here source of truth for the state of the UI that is not right :P please move this  variable to the viewmodel state object